src/metabase/models/recent_views.clj

```diff
@@ -1,9 +1,22 @@
 (ns metabase.models.recent-views
-  "The Recent Views table is used to track the most recent views of objects such as Cards, Tables and Dashboards for
-  each user."
+  "The Recent Views table is used to track the most recent views of objects such as Cards, Models, Tables, Dashboards,
+  and Collections for each user, for an up to date list, see [[models-of-interest]].
```

Comment:
grammar nit (or could just change the `,` to `;`)
```suggestion
  "The Recent Views table is used to track the most recent views of objects such as Cards, Models, Tables, Dashboards,
  and Collections for each user. For an up to date list, see [[models-of-interest]].
```

Reply:
Fixed